src/node_file.cc

```diff
@@ -273,8 +273,9 @@ struct fs_req_wrap {
     uv_req->result = err;                                                     \
     uv_req->path = nullptr;                                                   \
     After(uv_req);                                                            \
```

Comment:
Can you set `req_wrap = nullptr` here?

Aside: the `uv_req->path = nullptr` is a memory leak waiting to happen.  It's mostly benign now - I think.